app/src/main/java/com/android/sample/ui/utils/PlateSwipeScaffold.kt

```diff
@@ -100,7 +102,7 @@ private fun PlateSwipeTopBar(navigationActions: NavigationActions, showBackArrow
 
           Text(
               text = stringResource(id = R.string.plate_swipe_title),
-              style = MaterialTheme.typography.titleLarge,
+              style = MaterialTheme.typography.titleMedium,
```

Comment:
Please look at the PR about the theme corrections. I modify the font to be title medium and modify the overall topbar to be more consistent with the figma